### ISSUE TITLE
Advanced querying documentation

### ISSUE THREAD

**Issue by roll** [opentrials/opentrials#252]
I'm not sure but it looks like we don't have readme (preferably on the site) for doing advanced queries on Explorer.

**Comment by vitorbaptista** [comment]
For reference, the OpenFDA project also offers queries using ElasticSearch' query string. They have their documentation at https://open.fda.gov/api/

I've disabled the ElasticSearch query string in https://github.com/opentrials/opentrials/pull/233 in the Explorer-side (i.e. you can still use query strings directly through the API). This was to avoid users being able to do invalid queries (e.g. https://github.com/opentrials/opentrials/issues/227).

This has the drawback of limiting what the user can do. Maybe we could remove the escaping of `:`, so at least the user can use features like `public_title:Foo` or `_exists_:bar`. Or we could add some "unsafe" flag to disable escaping these elements...

**Comment by roll** [comment]
I mean for now may be it could be short doc in docs or readme mostly for people who will be working on data stuff for upcoming few month.

**Comment by pwalsh** [comment]
@vitorbaptista why it is marked as blocked?

**Comment by pwalsh** [comment]
@vitorbaptista FYI there are multiple requests for this in the feedback after the launch.

**Comment by vitorbaptista** [comment]
@pwalsh It's blocked on https://github.com/opentrials/opentrials/issues/288

**Comment by pwalsh** [comment]
WONTFIX - covered in other cards
